### PR TITLE
Popup Window Project

### PR DESCRIPTION
**Pull request by JuanDBta**
In this branch you can find the following changes:

Add JS Folder with the following files activated in the html script:
  mainproject_html_elements.js  Generate html elements dinamically
  all_html_elements.js  Generate html elements dinamically
  mobile_menu.js  Generate mobile menu window
  popup_allprojects_mobile.js Generate popup window in Mobile Version
  popup_allprojects_desktop.js Generate popup window in Desktop Version

Add <script> at the end of html file
Linters for JS

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -267,6 +269,67 @@ header {
...
+.description {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+}
+
+#closexdescription {
+  margin-left: 83vw;
+}
+
```
Comment (GedeonTS) [inline styles.css]: - I suggest positioning your popup as fixed to make sure the user can't lose it when he scrolls vertically :recycle: 
[Screencast from 20-04-2023 23:14:55.webm](https://user-images.githubusercontent.com/97834160/233489989-80f629c7-cdc4-4c7c-aa10-5ab23cc168ae.webm)

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @JuanDBta , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 

 - No linter errors :+1: 
- Good use of GitHub flow :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @JuanDBta & @diarisdiakite ✋

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly remove the entire live demo section from your readme file since this project is not deployed. This is a general requirement for all projects.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @JuanDBta  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [ ] Great work on implementing the required changes :+1: . However, I suggest blurring the background when the user opens the popup to improve the user experience. This will help to avoid any confusion and provide a clear focus on the popup content.

Screenshot from 2023-04-20 20-29-56


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**